src/common/http.cc

```diff
@@ -287,9 +288,18 @@ namespace Pistache::Http
             if (!match_until(' ', cursor))
                 return State::Again;
 
-            char* end;
-            auto code = strtol(codeToken.rawText(), &end, 10);
-            if (*end != ' ')
+            const char* beg = codeToken.rawText();
+            const char* beg_trim = beg;
+            while (*beg_trim == '+' || *beg_trim == ' ')
+            {
+                if (*(beg_trim + 1))
+                    ++beg_trim;                    
+            }
+            std::string_view view(beg_trim, codeToken.size());
+            const char* end = view.data() + view.size();
```

Comment:
Why is a string_view being used here?

Reply:
Citing cppreference:

>     constexpr basic_string_view( const CharT* s, size_type count );
>
> After construction, [data()](https://en.cppreference.com/w/cpp/string/basic_string_view/data) is equal to s, and [size()](https://en.cppreference.com/w/cpp/string/basic_string_view/size) is equal to count.

So isn't your code exactly the same as:

```suggestion
            const char* end = beg_trim + codeToken.size();
```